### PR TITLE
added latitude and longitude

### PR DESCRIPTION
**Pull request by VinnieLu**

### CODE REVIEW

**Comment by brianamarie** [comment]
Hi @VinnieLu! A few things to try to get the tests to pass:
- Make sure that `githubHandle` has a capital H for the word Handle
- Add these lines at the first line and last line of the document `---`, like the example 

Let me know if you need or want any help! 😄

File: _pins/vinnielu.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubhandle: Vinnielu
+latitude: 30.2643
```
Comment (VinnieLu) [inline _pins/vinnielu.json]: These are my coordinates

Comment (RussMorg-SMSDev) [inline _pins/vinnielu.json]: Hey @VinnieLu !

Seems like your json file is missing a few things. 😄 

The proper format is : 
+---  
 +githubHandle: <yourGitUserName> 
 +latitude: <latiNum>
 +longitude: <longNum>
 +---

Not far off though! 😄

**Comment by VladimirsPonomarjovs** [comment]
Wow. I believe this part is insanely good!!!!

**Pull request merged**